packages/crawler/src/page-processors/request-processor.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import Apify from 'apify';
+
+export interface RequestProcessor {
+    handleRequest(inputs: Apify.HandleRequestInputs): Promise<void>;
```

Comment:
Perhaps give this a name like handleSuccessfulRequest to make it symetrical with handleFailedRequest?

Reply:
I'm not sure if handleSuccessfulRequest would be an accurate name, handleFailedRequest is called on an error in handleRequest so there's no guarantee that the request is successful. Maybe renaming handleFailedRequest to handleError would be clearer?

Reply:
How about handleRequestError?